resources/patches/inox-patchset/chromium-clang-r4.patch

```diff
@@ -0,0 +1,11 @@
+--- a/device/fido/u2f_ble_transaction.cc
```

Comment:
So is this basically a newer version of the patch `ungoogled-chromium/linux/fix-libstdcxx630-errors.patch`? If so, we should probably remove that one.

And if I'm not mistaken, this is a fix for libstdc++, not clang. Using libc++ doesn't require this change (see any platform that isn't Debian or Ubuntu for version 65).

Reply:
If it's easier to keep things consistent with Inox, you can just remove my patch and rename yours to `fix-building-with-libstdcxx` or something, to prevent confusion with Gentoo's naming scheme.

If you aren't going to include this in Inox (since it's using libc++ now IIRC), then you can just refresh my patch and remove the `630` from the name (since it doesn't seem to be a problem with a specific libstdc++ version).

Reply:
It was not required and I refreshed and renamed your patch.